custom_import_map/public/plugin.tsx

```diff
@@ -3,14 +3,22 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
 import { i18n } from '@osd/i18n';
-import { CoreSetup, CoreStart, Plugin } from '../../../src/core/public';
+import React from 'react';
+import {
+  AppMountParameters,
+  CoreSetup,
+  CoreStart,
+  DEFAULT_APP_CATEGORIES,
+  Plugin,
+} from '../../../src/core/public';
 import {
-  CustomImportMapPluginSetup,
-  CustomImportMapPluginStart,
-  AppPluginSetupDependencies,
+  AppPluginStartDependencies,
+  MapServices, CustomImportMapPluginSetup, CustomImportMapPluginStart,
 } from './types';
+import { PLUGIN_NAME, PLUGIN_ID, PLUGIN_NAVIGATION_BAR_TILE } from '../common';
```

Comment:
I think this is import from '../common/constants/shared'

Reply:
fixed this and I have added the export statement in the index.ts file in that folder. So we can use both ../common and ../common/constants/shared